posix-configs/eagle/200qx/mainapp.config

```diff
@@ -1,9 +0,0 @@
-uorb start
```

Comment:
They are used, just not with our instructions. I tackle these soom but please leave them!